src/validation/enter.your.details.validation.ts

```diff
@@ -0,0 +1,26 @@
+import { body } from "express-validator";
+import { EnterYourDetailsErrorMessages } from "../utils/error_manifests/enter.your.details.errors";
+import { Address } from "private-api-sdk-node/src/services/presenter-account/types";
+
+const BlankValidator = {
+    premises: EnterYourDetailsErrorMessages.PREMISES_BLANK,
+    addressLine1: EnterYourDetailsErrorMessages.ADDRESS_LINE_1_BLANK,
+    townOrCity: EnterYourDetailsErrorMessages.TOWN_OR_CITY_BLANK,
+    postCode: EnterYourDetailsErrorMessages.POST_CODE_BLANK,
+    country: EnterYourDetailsErrorMessages.COUNTRY_BLANK
+};
```

Comment:
This is very misleading.
It should have a more descriptive name.
The name implies it is a validator but it is not. It just has the error messages for the fields.
Additionally, it is no different to the type `EnterYourDetailsErrorMessages` so I don't get it's purpose.